products/workers/src/content/runtime-apis/durable-objects.md

```diff
@@ -415,3 +416,10 @@ let response = await stub.fetch(url, options)
 The `fetch()` method of a stub has the exact same signature as the [global `fetch`](/runtime-apis/fetch). However, instead of sending an HTTP request to the internet, the request is always sent to the Durable Object to which the stub points.
 
 Any uncaught exceptions thrown by the Durable Object's `fetch()` handler are propagated to the caller's `fetch()` promise.
+
+## Listing Durable Objects
```

Comment:
This is notably *not* a runtime API (runtime APIs are the APIs that are natively exposed to Workers at runtime), which is what this page is all about. So it's a little weird to document it here instead of on the using-durable-objects page.

If you really want to keep it on this page, we should at least acknowledge that listing objects is only possible via `api.cloudflare.com`, not via a runtime API.

And if I were someone trying to use this section to figure out how to list my objects, I'd be disappointed when I found that hitting that URL directly didn't work if I didn't know which magic headers I have to provide for my request to be accepted. Are the standard API auth headers documented somewhere else that we could link to here?

Reply:
Whenever I've told people about the API I've always just said "see the top of https://api.cloudflare.com/ for Auth info" and (to my surprise?) it hasn't confused anyone yet.

Reply:
I agree this is an odd place to put it and linking to the top of https://api.cloudflare.com/ is enough for auth.

PR for the v4 api docs is at least merged to staging at this point though, so hopefully less of an issue

Reply:
this isn't a runtime API, but I would expect to see it called out on this page with a link to the actual docs.  I'll update.